.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: release
+on:
+  create:
+    tags:
+      - v*
+
+  workflow_dispatch:
```

Comment:
why would we want to manually dispatch official releases?

Reply:
Removed.